source/MulensModel/__init__.py

```diff
@@ -27,8 +28,4 @@
 for i in range(3):
     MODULE_PATH = path.dirname(MODULE_PATH)
```

Comment:
Is this variable necessary anymore?

Reply:
It was used in examples/plots_2.py which produced a plot for paper describing MM, so I'll leave it as is, so that this code works.